packages/rum-core/src/domain/rumSession.ts

```diff
@@ -16,8 +16,11 @@ export enum RumSessionPlan {
 
 export enum RumTrackingType {
   NOT_TRACKED = '0',
-  TRACKED_WITH_RESOURCES = '1',
-  TRACKED_WITHOUT_RESOURCES = '2',
+  // Note: the "tracking type" value (stored in the session cookie) does not match the "session
+  // plan" value (sent in RUM events). This is expected, and was done to keep retrocompatibility
+  // with active sessions when upgrading the SDK.
+  TRACKED_REPLAY = '1',
+  TRACKED_LITE = '2',
```

Comment:
As `TRACKED_WITHOUT_RESOURCES` is not strictly equivalent to `TRACKED_LITE`, we could also consider to perform a new draw or consider it as not tracked.
It sounds fine to me but it worth mentioning this behavior to @hdelaby.